### ISSUE TITLE
Use tags to alert Debian and other distros

### ISSUE THREAD

**Issue by eribertomota** [libyal/libpff#8]
Hi!

Please, use tags to create versions to allow the Debian and other distros can track new versions.

Thanks!

Cheers,

Eriberto

**Comment by joachimmetz** [comment]
Not sure what you mean. Isn't this automatically done by github releases?
Also do you have any documentation about the workflow you propose and how that "integrates" with the distros?

**Comment by eribertomota** [comment]
Hi! Thanks for your quick reply.

Yes, if you do a tag, the GitHub will create a release. However, you don't
have any tag/release now[1].

[1] https://github.com/libyal/libpff/releases

About a workflow, you need to put a tag only. The distros will use regex to
track all tags. You can see an example here[2].

[2] https://sources.debian.net/src/afflib/3.7.6-1/debian/watch/

Cheers,

Eriberto


2015-03-23 3:23 GMT-03:00 Joachim Metz <notifications@github.com>:

**Comment by joachimmetz** [comment]
In that nothing needs to change, the reason why you don't see a release is that there was none after the move to github. Hopefully soon if time permits.

**Comment by eribertomota** [comment]
There is the possibility of you tag in GitHub the last released version? (I
think that is 20120802)

Thanks


2015-03-23 11:55 GMT-03:00 Joachim Metz <notifications@github.com>:

**Comment by joachimmetz** [comment]
Does not make sense, the tag is linked to the state of the git repo. 20120802 is not a state in the git repo.

**Comment by eribertomota** [comment]
Yes. In git you can add a tag to a specif commit. An example:

$ git tag -am "Tagging version 0.01." 0.01 *e6a4f0707a*

Cheers,

Eriberto


2015-03-23 12:52 GMT-03:00 Joachim Metz <notifications@github.com>:

**Comment by joachimmetz** [comment]
I know but there is no such commit in the git repo ;)

**Comment by eribertomota** [comment]
Ah, ok. :-D

Yesterday I uploaded a new revision of the libpff to Debian. I used a
10-day delay[1] to wait for a reply for you. This revision says that the
package doesn't have releases. This is the only case without releases. So,
I will wait for a future release to change the watcher.

[1] https://ftp-master.debian.org/deferred.html

Thanks a lot for your work. I appreciate it.

Cheers,

Eriberto


2015-03-23 14:06 GMT-03:00 Joachim Metz <notifications@github.com>:
